.changeset/ninety-beans-compare.md

```diff
@@ -0,0 +1,16 @@
+---
+"@preact/signals": minor
+---
+
+Defer all DOM updates by an animation frame, this should make it so
+that any used to be synchronous DOM update will be delayed by an
+animation frame. This allows Preact to first perform its own render
+cycle and then our direct DOM updates to occur. These will now
+be performed in a batched way which is more performant as the browser
+is prepared to handle these during the animation frame.
+
+This does have one way to how Preact based signals are tested, when
```

Comment:
Sorry, not quite sure what's meant here but I think there's a typo. "This does impact how..." perhaps?

Reply:
```suggestion
This does impact how Preact based signals are tested, when
```